src/components/tr/tr-styles.jsx

```diff
@@ -31,22 +31,22 @@ export default theme => {
       '&.sticky': {
         position: 'absolute',
         background: palette.background.default,
-      },
-
-      [mixins.media('md')]: {
-        '&.sticky': {
-          position: 'fixed',
-        },
-
-        '& + .tr--clone': {
-          display: 'flex',
+        [mixins.media('md')]: {
+          '&.sticky': {
+            position: () => 'fixed',
```

Comment:
unnecessary function

Reply:
Needed due to dynamic stylesheet overrides static ones